modules/mako/module.nix

```diff
@@ -0,0 +1,57 @@
+{
+  wlib,
+  lib,
+  ...
+}:
+wlib.wrapModule (
+  { config, wlib, ... }:
+  let
+    iniFormat = config.pkgs.formats.iniWithGlobalSection { };
+    iniAtomType = iniFormat.lib.types.atom;
+  in
+  {
+    options = {
+      "config" = lib.mkOption {
```

Comment:
hmm, using "config" is a bit clunky, but it seems that is indeed the filename they give to the default config file? :D should we use something more descriptive maybe? but fine to merge it like this as well